### ISSUE TITLE
JSON RPC: admin_removePeer does not work as expected

### ISSUE THREAD

**Issue by MysticRyuujin** [NethermindEth/nethermind#1116]
`v1.2.6`

When calling `admin_removePeer` against a peer received by `admin_peers` the server returns the error `Static node was not found`. I assume this function should work against any peer, not just statically configured ones...?

```
PS > $Peer = Query-EthNode -Servers nethermind01 -Function admin_peers | Where-Object { $_.enode -match "e56dbb6cb7b3181335" }

# Show Peer Info

PS > $Peer

clientId   : Geth/v1.9.8-stable-d62e9b28/linux-amd64/go1.13.4
host       : 121.127.230.150
port       : 30305
address    : 121.127.230.150:30305
isBootnode : False
isTrusted  : False
isStatic   : False
enode      : enode://e56dbb6cb7b31813350cd31f9185322f8b198fa742b2aa6f5a46e70066c1a9974d9e0ef693fda1b031f58ee7dced3c52bbb46f32ac248bd04c8a4d937df03674@121.127.230.150:30305

# Use DebugJSON to show HTTP POST BODY

PS > Query-EthNode -Servers nethermind01 -Function admin_removePeer -enode $Peer.enode -DebugJSON

Name                           Value
----                           -----
method                         admin_removePeer
params                         {enode://e56dbb6cb7b31813350cd31f9185322f8b198fa742b2aa6f5a46e70066c1a9974d9e0ef693fda1b031f58ee7dced3c52bbb46f32ac248bd04c8a4d937df03674@121.127.230.150:30305}
id                             728087388
jsonrpc                        2.0

## Send admin_removePeer

PS > Query-EthNode -Servers nethermind01 -Function admin_removePeer -enode $Peer.enode
WARNING: Static node was not found.
```

This same code works against Geth

**Comment by tkstanczak** [comment]
@LukaszRozmej do you think you can have a look while I am away from the keyboard? This could add a peer to a 'banned' peers list in the same code area and just never allow connecting with them in PeerManager

**Comment by LukaszRozmej** [comment]
@tkstanczak In my opinion this works as designed. This feature was to mimic geth functionality. Geth allows only for removal of static nodes. It is not even documented in their API:

https://geth.ethereum.org/docs/rpc/ns-admin

But  can be found here:
https://github.com/ethereum/go-ethereum/pull/2740
https://ethereum.stackexchange.com/questions/6778/removing-a-peer-node

I would rather keep feature parity with geth here not to confuse users with same name but different behavior.

Another thing is this might turn into feature request to add something like 'ignored nodes' that would act as a blacklist. This is something different altogether.

**Comment by MysticRyuujin** [comment]
@LukaszRozmej admin_removePeer works perfectly fine with Geth as stated at the bottom of my issue.

Here's the proof:

```
PS > (Query-EthNode -Servers geth01 -Function admin_nodeInfo).name
Geth/MysticRyuujin-geth01/v1.9.10-unstable-370cb95b-20191206/linux-amd64/go1.13.4
PS > $Peer = Query-EthNode -Servers geth01 -Function admin_peers | Where-Object { $_.enode -match "11c919581560bd5994b6323958c3b9414455ae358"}
PS > $Peer


enode     : enode://11c919581560bd5994b6323958c3b9414455ae358acfe8d00be2e2f09c32b0463f09a05f3b01b6298496d01e89432248309e68cbf65d3f142a63fb1ee913ad7b@157.230.125.19:30303
id        : b6668cfe95facbe05a4266a65b26ea33553df49e50d2c63ee9ed05d0e3df5e9c
name      : Geth/v1.9.8-stable-d62e9b28/linux-amd64/go1.13.4
caps      : {eth/63, eth/64}
network   : @{localAddress=192.168.15.31:38550; remoteAddress=157.230.125.19:30303; inbound=False; trusted=False; static=False}
protocols : @{eth=}



PS > Query-EthNode -Servers geth01 -Function admin_removePeer -enode $Peer.enode
True
PS > Query-EthNode -Servers geth01 -Function admin_peers | Where-Object { $_.enode -match "11c919581560bd5994b6323958c3b9414455ae358"}
PS >
```

**Comment by tkstanczak** [comment]
@LukaszRozmej I would definitely trust users here rather than specs - I think @MysticRyuujin is practically an expert here
I will have a look, I still believe that the fix will be simple.

I will probably only be able to get it merged around Thursday.

**Comment by tkstanczak** [comment]
Fixed, will be released in 1.3.3
